[2021-04-04 22:31:59,565] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-04-05T02:27:23.398734+00:00 [queued]>
[2021-04-04 22:31:59,568] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-04-05T02:27:23.398734+00:00 [queued]>
[2021-04-04 22:31:59,568] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-04 22:31:59,569] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-04 22:31:59,569] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-04 22:31:59,599] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): processing_user> on 2021-04-05T02:27:23.398734+00:00
[2021-04-04 22:31:59,602] {standard_task_runner.py:52} INFO - Started process 3501 to run task
[2021-04-04 22:31:59,608] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2021-04-05T02:27:23.398734+00:00', '--job-id', '21', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpdxy_55w9', '--error-file', '/tmp/tmp2ayd9rg_']
[2021-04-04 22:31:59,609] {standard_task_runner.py:77} INFO - Job 21: Subtask processing_user
[2021-04-04 22:31:59,637] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2021-04-05T02:27:23.398734+00:00 [running]> on host prathamesh-VirtualBox
[2021-04-04 22:31:59,669] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-04-05T02:27:23.398734+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-04-05T02:27:23.398734+00:00
[2021-04-04 22:31:59,682] {python.py:118} INFO - Done. Returned value was: None
[2021-04-04 22:31:59,684] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20210405T022723, start_date=20210405T023159, end_date=20210405T023159
[2021-04-04 22:31:59,707] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-04 22:31:59,729] {local_task_job.py:146} INFO - Task exited with return code 0
